### ISSUE TITLE
Non-linear references

### ISSUE THREAD

**Issue by timwis** [gera2ld/markmap-lib#24]
Hi! Really cool tool. Is there any support for non-liner references? For example when a grandchild of A has a connection to a child of B, or an item is connected to multiple parents? Basically, drawing a connection between two nodes that are not in the same branch.

```markdown
# Goals

## Open new branch
- Close property deal
- Hire additional team members [connects to Secure new investment]

## Secure new investment
- Prepare business plan
- Find investors
```

**Comment by gera2ld** [comment]
This sounds good, but I'm not sure how to visualize the unpredicted connections.

**Comment by timwis** [comment]
Do you mean aesthetically? I’d assume it’s just another line between the two nodes. Or do you mean how to identify which node to connect to without IDs?

**Comment by gera2ld** [comment]
> Do you mean aesthetically?

Yep, I meant it. I think the lines should not strike through text.

**Comment by timwis** [comment]
How about a faded dashed line connecting them that, when you hover on it, increases its opacity and emphasises the nodes it connects?
